Wrapping/Generators/SwigInterface/igenerator.py

```diff
@@ -608,17 +608,32 @@ def generate_process_object_snake_case_functions(self, typedefs):
                 self.outputFile.write('    import itk\n')
                 self.outputFile.write('    import itkTemplate\n')
                 self.outputFile.write('    import itkHelpers\n')
+                self.outputFile.write('    specific_args, exclude_args, common_args = None, None, None\n')
+                self.outputFile.write('    exclude_args = [itkHelpers.camel_to_snake_case(item[3:]) for item in dir(itk.Object) if item[:3] == "Set"]\n')
+                self.outputFile.write('    common_args =  [itkHelpers.camel_to_snake_case(item[3:]) for item in dir(itk.ProcessObject) if item[:3] == "Set"]\n')
```

Comment:
Let's put these in the `itkHelpers.py` module so we do not have to re-compute them for every function.